tools/pipelines/register-es-pipelines.js

```diff
@@ -61,37 +61,94 @@ const GEONETWORK_UI_PIPELINE = {
       script: {
         lang: 'painless',
         source: `
-int total=8;
+int total=0;
```

Comment:
We should probably initialize this with another value than 0 (or do a check at the end) to prevent division by zero at the end of the pipeline if there is a `resourceType` we do not detect in the script. This is a case I stumbled upon before the commit for `featureCatalog`.